db/seeds.d/020-roles_list.rb

```diff
@@ -26,7 +26,7 @@ def seeded_roles
                                              :view_smart_proxies, :edit_smart_proxies, :view_subnets, :edit_subnets,
                                              :view_statistics, :view_usergroups, :create_usergroups, :edit_usergroups,
                                              :destroy_usergroups, :view_users, :edit_users, :view_realms, :view_mail_notifications,
-                                             :view_params, :view_ssh_keys, :view_personal_access_tokens],
+                                             :view_params, :view_ssh_keys, :view_personal_access_tokens, :access_user_columns],
```

Comment:
This should be added to viewer/manager/org admin roles. Or even better, it shoud be part of default role, there's no point in not having this permission for any user.

Reply:
> This should be added to viewer/manager/org admin roles. Or even better, it shoud be part of default role, there's no point in not having this permission for any user.

Thanks for the pointer. Updated :+1: